GETTING_STARTED.md

```diff
@@ -34,6 +34,16 @@ It's also worth checking your migrations in a separate, containerized environmen
 
 **IMPORTANT:** If you would like to run multiple instances of your app on different machines be sure to use a database that supports locking when running migrations. Otherwise you may encounter issues.
 
+## Forcing your database version
+In case you run a migration that contained an error, migrate will not let you run other migrations on the same database. You will see an error `Dirty database version 1. Fix and force version`, even when you fix migration. This means your database was marked as 'dirty'.
+You need to investigate the migration error - was your migration applied partially, or was it not applied at all? Once you know, you should force your database to a version reflecting it's real state. You can do so with `force` command:
+```
+migrate -path PATH_TO_YOUR_MIGRATIONS -database YOUR_DATBASE_URL force VERSION
+```
+Once you force the version and your migration was fixed, your database is 'clean' again and you can proceed with your migrations.
+
+For details and example of usage see [this comment](https://github.com/golang-migrate/migrate/issues/282#issuecomment-530743258).
```

Comment:
I was not sure if adding a link to a comment in an issue is ok, I have just felt like it's already well described there and I wanted to give credit where it's due.
If you want me to rewrite it, that's also fine.